sdk/identity/Azure.Identity.Broker/src/InteractiveBrowserCredentialBrokerOptions.cs

```diff
@@ -25,6 +25,11 @@ public class InteractiveBrowserCredentialBrokerOptions : InteractiveBrowserCrede
         /// </summary>
         public bool IsProofOfPossessionRequired { get; set; }
 
+        /// <summary>
+        /// Authenticate with the currently signed in user instead of prompting the user with a login dialog.
```

Comment:
Nit (and in other files): for consistency both in this API and with .NET general docs,
```suggestion
        /// Gets or sets whether to authenticate with the currently signed in user instead of prompting the user with a login dialog.
```